src/main/java/org/folio/circulation/resources/RequestFromRepresentationService.java

```diff
@@ -578,6 +578,12 @@ private Request removeRelatedRecordInformation(Request request) {
     representation.remove("pickupServicePoint");
     representation.remove("deliveryAddress");
 
+    JsonObject printDetails = representation.getJsonObject("printDetails");
+    if (printDetails != null && printDetails.containsKey("lastPrintRequester")) {
+      printDetails.remove("lastPrintRequester");
+    }
+
+
```

Comment:
Do we need two blank lines here?

Reply:
fixed: https://github.com/folio-org/mod-circulation/pull/1492/commits/597143297af4b5344f832f462d339b4dbb58dd3c